config/gnbsim.yaml

```diff
@@ -48,7 +48,7 @@ configuration:
         mnc: 93 # Mobile Network Code (2 or 3 digits string, digit: 0~9)
     - profileType: pdusessest # profile type
       profileName: profile2 # uniqely identifies a profile within application
-      enable: true # Set true to execute the profile, false otherwise.
+      enable: false # Set true to execute the profile, false otherwise.
```

Comment:
So by default no profiles will be enabled if we pick this configuration file.

Reply:
This is just example configuration. Actual config is coming from override files.